pylintrc

```diff
@@ -10,9 +10,9 @@
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
-#disable=no-absolute-import,bad-continuation,C0103,C0114,C0115,E1101,W0201,no-name-in-module
-disable=all
-enable=redefined-builtin,useless-object-inheritance,trailing-newlines
+disable=no-absolute-import,bad-continuation,C0103,C0114,C0115,E1101,W0201,no-name-in-module,W0613,C0116
+#disable=all
+#enable=redefined-builtin,useless-object-inheritance,trailing-newlines
```

Comment:
Maybe we should add missing-docstring in there because with pylint 2.3.1 I see a lot of errors like this:
```
************* Module portage.tests.util.test_file_copier
lib/portage/tests/util/test_file_copier.py:1:0: C0111: Missing module docstring (missing-docstring)
lib/portage/tests/util/test_file_copier.py:15:0: C0111: Missing class docstring (missing-docstring)
```